src/Form/AdminSettingsForm.php

```diff
@@ -317,6 +318,31 @@ private function buildThumbnailUrlForm() {
   }
 
   /**
+   * Display advanced configurations forms.
+   *
+   * @return array
+   *   The render array for the advanced configuration form.
+   */
+  private function buildAdvancedConfigurationForm() {
+    $settings = $this->config('acquia_lift.settings');
+
+    $form = [
+      '#title' => t('Advanced configuration'),
+      '#type' => 'details',
+      '#tree' => TRUE,
+      '#open' => FALSE,
+    ];
+    $form['content_replacement_mode'] = [
+      '#type' => 'radios',
+      '#title' => t('Content replacement mode'),
+      '#default_value' => $settings->get('content_replacement_mode'),
+      '#options' => ['trusted' => t('Trusted'), 'untrusted' => t('Untrusted')],
+    ];
+
+    return $form;
+  }
```

Comment:
I would personally recommend the following changes to "Advanced" tab:

1. In admins menu, there is already a "Data collection settings", which encapsulates all the data collection settings. We are actually introducing its sibling, "Content deployment settings", for the first time.
2. Rename "Advanced" to "Content", and list as the first tab under "Content deployment settings".
3. "Content" tab should have only one radio buttons set, which is "Default content deployment mode" of trusted / untrusted.
4. Add descriptions on trusted / untrusted mode; make aware this is a _default_ and _override-able_ value.

The reasons are:
* My old plan was to tree-group all the "data collection" options in one tab list, and the "content deployment" in another tab list.  It would be nice if I get to keep this plan, because at least two nice things come out of it:
  * Clear division and easy to find.
  * In the distance future when this pages grows too large, we split it in to 3 pages "Credential", "Data Collection", and "Content deployment".
* Choosing the default trusted mode doesn't seem to be "advanced" to me. This is one of the most requested feature as per late feedbacks, and I would consider it essential, and would expect it to be one of the first items that an admin would happily change to.
* It is the best place to describe trusted / untrusted mode because an Admin sees this page before using the Experience Builder, and this setting has a serious impact on their personalizations' implementation.